keras_cv/losses/centernet_box_loss.py

```diff
@@ -61,40 +60,43 @@ class CenterNetBoxLoss(keras.losses.Loss):
     def __init__(self, num_heading_bins, anchor_size, **kwargs):
         super().__init__(**kwargs)
         self.num_heading_bins = num_heading_bins
-        self.anchor_size = anchor_size
+        self.anchor_size = ops.convert_to_tensor(anchor_size)
```

Comment:
this is curious, why the need to convert? I have been trying to avoid passing tensor types as init args.

Reply:
Good catch -- I needed this earlier due to some PyTorch device weirdness, but it got fixed in Keras Core so this is no longer needed.